### PR TITLE
Add new pricing plans

### PR DESCRIPTION
**Pull request by bcaudan**
Update `_dd.session.plan` with new pricing codes

### CODE REVIEW

**Review by BenoitZugmeyer — APPROVED** [review]

**Review by liywjl — APPROVED** [review]

**Review by xgouchet — APPROVED** [review]

File: lib/cjs/generated/rum.d.ts [hunk 1]
```diff
@@ -858,9 +858,9 @@ export interface CommonProperties {
          */
         session?: {
             /**
-             * Session plan: 1 is the 'lite' plan, 2 is the 'replay' plan
+             * Session plan: 1 is the 'lite' plan, 2 is the 'premium' plan, 3 is the 'pro' plan, 4 is the 'replay' plan
              */
-            plan: 1 | 2;
+            plan: 1 | 2 | 3 | 4;
```
Comment (ThibautGeriz) [inline lib/cjs/generated/rum.d.ts:863]: **question:** replay was 2 and now it's 4, are we sure about this? It won't break existing behaviour?
**Disclaimer:** I have no idea what the new plans are

Comment (bcaudan) [reply lib/cjs/generated/rum.d.ts:863]: The only impact here is on the comment.
'replay' for 2 was outdated and we never fixed it to premium.
To be noted: 'pro' and 'replay' namings may still change, not a big deal for the format structure though.

**Review by acorretti — APPROVED** [review]

File: lib/cjs/generated/rum.d.ts (hunk 1 above)
Comment (benoitmeriaux) [reply lib/cjs/generated/rum.d.ts:863]: 2 should be labeled as "premium" ? we never had this value in the backend 😅, it is "replay" everywhere, even in the "nicely formatted" field for customers

**Review by benoitmeriaux — APPROVED** [review]

File: lib/cjs/generated/rum.d.ts (hunk 1 above)
Comment (bcaudan) [reply lib/cjs/generated/rum.d.ts:863]: it should have been but a bit late now 🙂

**Pull request merged**